src/lamp_py/bus_performance_manager/write_events.py

```diff
@@ -50,8 +61,18 @@ def write_bus_metrics() -> None:
                     extra_args={"Metadata": {VERSION_KEY: bus_events.version}},
                 )
 
-            day_logger.log_complete()
+            # if any day succeeds, flip true - triggers upload to tableau
+            successful_metrics = True
+
+        except (LampExpectedNotFoundError, LampInvalidProcessingError):
```

Comment:
The way these exceptions are currently thrown, `LampInvalidProcessingError` means something unexpected has gone wrong with the process. This should not be continued on, the error should be logged.